extensions/3rdparty/LyricWiki/LyricFind/LyricFind.setup.php

```diff
@@ -31,12 +31,6 @@
 // LyricFind indexing
 $wgHooks['ParserBeforeStrip'][] = 'LyricFindHooks::onParserBeforeStrip';
 
-// edit permissions & view-source protection
-// @see http://www.mediawiki.org/wiki/Manual:$wgNamespaceProtection
-$wgGroupPermissions['*']['editlyricfind'] = false;
-$wgGroupPermissions['staff']['editlyricfind'] = true;
-$wgGroupPermissions['sysop']['editlyricfind'] = true;
-
 $wgHooks['AlternateEdit'][] = 'LyricFindHooks::onAlternateEdit';
```

Comment:
Please find out if we can remove everything related to LyricFind. AFAIK we stopped using that service X years ago.
@owend @SeanColombo